utils/bump_cwl_version.py

```diff
@@ -13,74 +15,22 @@
 ERROR_MISSING_DOCKER_REQUIREMENT = 'CWL tool is missing DockerRequirement'
 ERROR_MISSING_DOCKER_PULL = 'Please specify "dockerPull" in your DockerRequirement and rerun script'
 
+
 def tools_list(tools_dir):
   log.debug(f'tools_dir passed to tools_list={tools_dir}')
-  glob_pattern = f'{tools_dir}/*.cwl'
+  glob_pattern = f'{tools_dir}/*.mustache'
   log.debug(f'glob_pattern ={glob_pattern}')
   return glob.glob(glob_pattern)
 
 
-def read_tool(path):
-  with open(path) as file:
-    tool = yaml.load(file, Loader=yaml.FullLoader)
-  return tool
-
-
-def parse_docker_image(tool):
-
-  def find_docker_requirement(object):
-    if type(object) is dict:
-      docker_requirement = object.get('DockerRequirement', None)
-      object_type = dict
-    elif type(object) is list:
-      docker_requirement = next((item for item in object if item['class'] == 'DockerRequirement'), None)
-      object_type = list
-    else:
-      raise ValueError(ERROR_UNEXPECTED_TYPE)
-
-    return docker_requirement,object_type
-
-  docker_requirement = None
-
-  if 'hints' in tool:
-    docker_requirement, object_type = find_docker_requirement(tool['hints'])
-    if docker_requirement is not None:
-      parsing_metadata = {
-        'docker_requirement_found_in': 'hints',
-        'object_type': object_type
-      }
-
-  if docker_requirement is None and 'requirements' in tool:
-    docker_requirement, object_type = find_docker_requirement(tool['requirements'])
-    if docker_requirement is not None:
-      parsing_metadata = {
-        'docker_requirement_found_in': 'requirements',
-        'object_type': object_type
-      }
-
-  if docker_requirement is None:
-    raise ValueError(ERROR_MISSING_DOCKER_REQUIREMENT)
-
-  docker_image = docker_requirement.get('dockerPull', None)
-
-  if docker_image is None:
-    raise ValueError(ERROR_MISSING_DOCKER_PULL)
-
-  return docker_image, parsing_metadata
-
-
-def edit_tool(tool, new_version):
-  docker_image, parsing_metadata = parse_docker_image(tool)
-  parts = docker_image.split(':')
-  parts[-1] = new_version
-  tool_obj = tool[parsing_metadata['docker_requirement_found_in']]
-  new_docker_image = ':'.join(parts)
-  if parsing_metadata['object_type'] is dict:
-    docker_requirement = tool_obj['DockerRequirement']
-  else:
-    docker_requirement = next((item for item in tool_obj if item['class'] == 'DockerRequirement'))
-  docker_requirement['dockerPull'] = new_docker_image
-  return yaml.dump(tool)
+def create_tool(template_path, new_version, tools_dir):
+  cwl_input = {'version': new_version}
+  tool_name = os.path.basename(template_path).replace(".mustache", "")
```

Comment:
[Nit:](https://google.github.io/eng-practices/review/reviewer/standard.html#:~:text=Reviewers%20should%20always%20feel%20free,they%20could%20choose%20to%20ignore.) You don't need to do this here because I'm not worried about this matching anything other than the suffix, but for future reference, there is a `str.removesuffix()` method.

Reply:
Cool! Ill keep note that this function was added in python 3.9.